src/sass/components/_message_media.scss

```diff
@@ -1,10 +1,8 @@
 // Config variables
-
 $circle-size: 50px;
 $loading-ring-thickness: 5px;
 
 // Animations
-
```

Comment:
Hm, this removal sucks a bit, since the comment is not specifically for the next block, but for all folowing blocks until a new "section" starts. But I guess we can't configure the linter to consider that?

Reply:
Nah, these are the downsides of linting I guess.